rocky/reports/templates/partials/report_setup_scan.html

```diff
@@ -92,10 +92,17 @@ <h3>{% translate "Suggested plugins" %}</h3>
                     </div>
                 </div>
             {% endif %}
-            <form id="generate_report" class="inline" method="post" action="{{ next }}">
-                {% csrf_token %}
+            <form id="continue-to-configuration"
+                  class="inline"
+                  method="get"
+                  action="{{ next }}">
+                {% include "forms/report_form_fields.html" %}
+
+                {% for report_type in selected_report_types %}
+                    <input type="hidden" name="report_type" value={{ report_type }}>
```

Comment:
The attribute value should be enclosed with double quotes

Reply:
Since we are returning a list of these report_type inputs (one for each selected report), we should probably name this input report_type[] 
Also, do we need to specify each individually, or are we just re-posting the list we collected on an earlier view? If so, just posting the list as CSV would probably work.